### PR TITLE
README: add Cinnamon portal

### PR DESCRIPTION
**Pull request by nielsdg**
Cinnamon recently resolved a request [1]  to have a portal backend for their DE by introducing xdg-desktop-portal-xapp. So let's add that to the list of known portal backendss

[1]: https://github.com/linuxmint/cinnamon-desktop/issues/202
[2]: https://github.com/linuxmint/xdg-desktop-portal-xapp

### CODE REVIEW

**Comment by matthiasclasen** [comment]
I have my doubts that we want to keep this the canonical list of all available backend implementations, but we can add this one, for sure.

Maybe reword it slightly to make it clear that this list isn't necessarily exhaustive, say

    Here are some examples of available backends:

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Different backends are available see:
 - Pantheon (Elementary) [xdg-desktop-portal-pantheon](https://github.com/elementary/portals)
 - wlroots [xdg-desktop-portal-wlr](https://github.com/emersion/xdg-desktop-portal-wlr)
 - Deepin [xdg-desktop-portal-dde](https://github.com/linuxdeepin/xdg-desktop-portal-dde)
+- Cinnamon [xdg-desktop-portal-xapp](https://github.com/linuxmint/xdg-desktop-portal-xapp)
```
Comment (tristan957) [inline README.md:41]: Calling it the Xapp portal would probably be more correct instead of Cinnamon since XFCE and MATE are supported in the same codebase.

Comment (nielsdg) [reply README.md]: Fixed by using Simon's wording. Thanks!
